pkg/api/types/builder_types.go

```diff
@@ -82,4 +82,6 @@ type BuilderPruneOptions struct {
 	BuildKitHost string
 	// All will remove all unused images and all build cache, not just dangling ones
 	All bool
+	// Force will not prompt for confirmation.
+	Force bool
```

Comment:
Where is this consumed?

Reply:
Looks like it ended up being used on the CLI side only, so not much reason to pass it to the package backend.